src/components/pages/Home/Onboarding/Header.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ReactElement } from 'react'
+import { graphql, useStaticQuery } from 'gatsby'
+import Container from '../../../atoms/Container'
+import Markdown from '../../../atoms/Markdown'
+import styles from './Header.module.css'
+
+const query = graphql`
+  query OnboardingQuery {
+    file(relativePath: { eq: "pages/index/onboarding/index.json" }) {
+      childOnboardingJson {
+        title
+        subtitle
+        body
+      }
+    }
+  }
+`
+
+interface OnboardingHeaderData {
+  file: {
+    childOnboardingJson: {
+      title: string
+      subtitle: string
+      body: string
+    }
+  }
+}
+
+export default function Header(): ReactElement {
+  const data: OnboardingHeaderData = useStaticQuery(query)
+  const { title, subtitle, body } = data.file.childOnboardingJson
+
+  return (
+    <Container className={styles.container}>
+      <div className={styles.content}>
+        <h5 className={styles.subtitle}>{subtitle}</h5>
+        <h2 className={styles.title}>{title}</h2>
```

Comment:
nitpick(blocking): Previously title was treated as `h3`. Is there a way to unify this? I am asking since the subtitle is still `h5` here, but if you'd followed the previous logic it should be `h4`.

Reply:
My impression is that in the whole repo we are using the <h> tags more for the size than for the meaning of the tags. We also have some css global rules to modify the size of the <h> tags based on the screen size, if we use an <h3> instead of an <h2> then we need to "force it" to a different font size in css and put the media query inside the specific component. We can do it, it's just probably impractical to keep updated in the future